lib/tmuxinator/window.rb

```diff
@@ -34,13 +34,16 @@ def initialize(window_yaml, index, project)
 
     def build_panes(panes_yml)
       Array(panes_yml).map.with_index do |pane_yml, index|
-        if pane_yml.is_a?(Hash)
-          pane_yml.map do |_name, commands|
-            Tmuxinator::Pane.new(index, project, self, *commands)
-          end
-        else
-          Tmuxinator::Pane.new(index, project, self, pane_yml)
-        end
+        commands =  case pane_yml
+                    when Hash
+                      pane_yml.values.first
```

Comment:
Is there a situation in which there would be multiple elements in this Hash? As in, might the hash contain something like this?
```ruby
{ "one" => ["cd tmp", "vim"], "two" => ["cd log", "tail -f development.log"] }
```

If so, I think the easy fix is just changing this line to `pane_yml.values`